content/docker-hub/general-faqs.md

```diff
@@ -1,54 +1,50 @@
+title: General FAQs for Docker accounts
+description: Frequently asked Docker account and administration questions
+keywords: onboarding, docker, teams, orgs, user accounts, organization accounts
 redirect:
 - /docker-hub/onboarding-faqs/
 ---
 
 ### What is a Docker ID?
 
 A Docker ID is a username for your Docker account that lets you access Docker products. All you need is an email address to create a Docker ID, or you can sign up with your Google or GitHub account. Your Docker ID must be between 4 and 30 characters long, and can only contain
-numbers and lowercase letters. You cannot use any special characters or spaces.
+numbers and lowercase letters. You can't use any special characters or spaces.
 
-For more information, see [Docker ID](../docker-id/index.md). If your admin enforces [Single sign-on (SSO)](../security/for-admins/single-sign-on/index.md), a Docker ID is provisioned for new users.
+For more information, see [Docker ID](../docker-id/index.md). If your administrator enforces [Single sign-on (SSO)](../security/for-admins/single-sign-on/index.md), this provisions a Docker ID for new users.
 
-Developers may have multiple Docker IDs in order to separate their Docker IDs that are associated with an organization in Docker Business or Team, and their personal use Docker IDs.
+Developers may have multiple Docker IDs in order to separate their Docker IDs associated with an organization with a Docker Business or Team subscription, and their personal use Docker IDs.
 
 ### What if my Docker ID is taken?
 
 All Docker IDs are first-come, first-served except for companies that have a US Trademark on a username. If you have a trademark for your namespace, [Docker Support](https://hub.docker.com/support/contact/) can retrieve the Docker ID for you.
 
 ### What’s an organization?
 
-Docker users become members of an organization when they're associated with the organization by an organization owner. An organization owner is someone assigned the owner role. They
-can create new teams and add members to an existing team using their Docker ID
-or email address and by selecting a team the user should be part of. An
-organization owner can also add additional organization owners to help them
-manage users, teams, and repositories in the organization. See [Create your organization](orgs.md).
+An organization in Docker is a collection of teams and repositories that administrators can manage together. Docker users become members of an organization once they're associated with that organization by an organization owner. An organization owner is a user with administrative access to the organization. They can create new teams and add members to an existing team using their Docker ID or email address and by selecting a team the user should be part of. An organization owner can also add additional organization owners to help them manage users, teams, and repositories in the organization. See [Create your organization](orgs.md).
```

Comment:
same issue in some of these sentences where the subject becomes ambiguous

Reply:
Reworded and removed some of this content and linked to the **Who is an organization owner?** question as this was repetitive and was really addressing that question.